packages/demo/compat/glimmer-util.ts

```diff
@@ -0,0 +1,14 @@
+export function unwrapTemplate(tpl) {
+  return {
+    asLayout(){
```

Comment:
I could be misremembering the purpose of this...
But, I don't think layouts need support as template resolving outside of routes' templates is deprecated for removal in v6

Reply:
for now it's workaround related to existing template handing logic (from glimmer-vm).
compiled glimmer-vm template has special 'shape' and it's hardcoded in ember - we just providing compatible wrapper for it.